legal-api/report-templates/certificateOfIncorporation.html

```diff
@@ -24,8 +24,8 @@
 
           <div class="certify-clause-container">
             <div class="cert-statement">I hereby certify that</div>
-            <div class="company-name-header">{{ business.legalName }}</div>
-            <div class="cert-statement">was incorporated under the Business Corporations Act</div>
+            <div class="company-name-header"><div class="company-name-text">{{ business.legalName }}</div></div>
```

Comment:
Can nested div's be on new lines for easier readability?

Also, why not combine the classes and have a single div?

Reply:
I will make this fix in another PR for an API change